conjure/controllers/bootstrapwait/gui.py

```diff
@@ -0,0 +1,29 @@
+from conjure.ui.views.bootstrapwait import BootstrapWaitView
+from conjure.app_config import app
+from conjure import controllers
+from ubuntui.ev import EventLoop
+import sys
+
+this = sys.modules[__name__]
+
+
+def finish(*args):
+    return controllers.use('deploystatus').render()
+
+
+def __refresh(*args):
+    this.view.redraw_kitt()
+    EventLoop.set_alarm_in(1, __refresh)
```

Comment:
I think you need to store the alarm this returns and cancel it in finish(), because otherwise we'll just redraw kitt forever even after we've set the body to a different view